### PR TITLE
Minor adjustments to the brand intelligence event (and also a tiny change to stray meteor admin setup)

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

The events folder is crusty, and I've spent some past PRs in the folder already, so I figured why not just run down the line and make sure everything's up to modern standards (or at least my personal, likely underwhelming standards). First up is a tried and true classic, clocking in at over a decade old -- Brand Intelligence. 

Here's what's been changed:

* Admin setup functionality has been added. You can now pick which vendor subtype will be selected as patient zero. I'm not sure how much use this will see, but admin_setup is disappointingly underused and should (will) be expanded to more events. This also makes a minor change to the stray meteor admin setup message because I noticed that I fucking _set it up backwards_ in my last PR while testing for this one. I hope this doesn't count as being too out-of-scope for this PR.

* False Alarm no longer selects a cola vendor as its reported patient zero. It now selects its announcement from an actual on-station vendor.

* Modified the announcement slightly. It's hopefully not too wordy, but I wanted the announcement to give a bit more direction on how to proceed when it fires.

* Translates some leftover binaries in vending (that were only affected by the brand intelligence event) to use TRUE/FALSE instead of 1/0

* Smites the last single character varname in the file out of existence.

Nothing to note outside of that. Flawless event.

## Why It's Good For The Game

Slightly less crusty code, more believable false alarms. Increasing admin setup implementation (and fixing my old code) is good.

The announcement being changed should hopefully give a bit more guidance on what the brand intelligence event expects from the crew. Less overtly harmful events like carp migration can get away with having a more oblique announcement message, but an event as potentially disastrous as this one should be a bit more direct. Now players should be able to at least glean what the event is asking of them without having to search through the wiki or the github page.

## Changelog

🆑
fix: Brand Intelligence false alarm will no longer only report that a cola vendor has been hit, and will instead use an on-station vendor to lie to players more accurately.
spellcheck: Modified the brand intelligence announcement slightly to give a bit more direction to the crew.
code: Changed some vendor vars to use TRUE/FALSE instead of 1/0
code: Murders the last remaining single character varname in the brand intelligence file.
admin: Brand Intelligence event now supports admin setup. Choose your brand of brand intelligence!
admin: Stray meteor event admin setup message no longer lies to you.
/🆑

### CODE REVIEW

File: code/modules/events/brand_intelligence.dm [hunk 1]
```diff
@@ -7,7 +7,16 @@
 
 	min_players = 15
 	max_occurrences = 1
+	//Var used to determine vendor subtype if used for admin setup
+	var/chosen_vendor
```
Comment (san7890) [inline code/modules/events/brand_intelligence.dm:11]:
```suggestion
	///Var used to determine vendor subtype if used for admin setup
	var/chosen_vendor
```

just so it can DMDoc

File: code/modules/events/brand_intelligence.dm [hunk 2]
```diff
@@ -28,30 +37,42 @@
...
+		var/chosen_vendor = brand_event.chosen_vendor
+		for(var/obj/machinery/vending/vendor in GLOB.machines)
+			if(!is_station_level(vendor.z) || !istype(vendor, chosen_vendor))
+				continue
+			vendingMachines.Add(vendor)
+	else
+		for(var/obj/machinery/vending/vendor in GLOB.machines)
+			if(!is_station_level(vendor.z))
+				continue
+			vendingMachines.Add(vendor)
```
Comment (JohnFulpWillard) [inline code/modules/events/brand_intelligence.dm:62]: instead of this being an else, it should instead checkfor ``length(vendingMachines)``, that way it will only run if there's no vending machines in the list already (in case admins choose a vending machine that doesn't exist)
